tests/plugins/test_koji_import.py

```diff
@@ -2101,3 +2128,164 @@ def test_set_operators_metadata(self, tmpdir, os_env, has_manifests, reactor_con
             assert extra['operator_manifests']['appregistry'] == PUSH_OPERATOR_MANIFESTS_RESULTS
         else:
             assert 'operator_manifests' not in extra
+
+    @pytest.mark.parametrize('blocksize', (None, 1048576))
+    @pytest.mark.parametrize('has_config', (True, False))
+    @pytest.mark.parametrize('tag_later', (True, False))
+    @pytest.mark.parametrize(('verify_media', 'expect_id'), (
+        (['v1', 'v2', 'v2_list'], 'ab12'),
+        (['v1'], 'ab12'),
+        (False, 'ab12')
+    ))
+    @pytest.mark.parametrize('reserved_build', (True, False))
+    def test_koji_import_success_source(self, tmpdir, blocksize, os_env, has_config,
+                                        tag_later, verify_media, expect_id,
+                                        reserved_build, reactor_config_map):
+        session = MockedClientSession('')
+        # When target is provided koji build will always be tagged,
+        # either by koji_import or koji_tag_build.
+        component = 'component'
+        name = 'ns/name'
+        version = '1.0'
+        release = '1'
+
+        tasker, workflow = mock_environment(tmpdir,
+                                            session=session,
+                                            name=name,
+                                            component=component,
+                                            version=version,
+                                            release=release,
+                                            has_config=has_config,
+                                            source_build=True)
+
+        workflow.build_result = BuildResult(oci_image_path="oci_path")
+        workflow.koji_source_nvr = {'name': name, 'version': version, 'release': release}
```

Comment:
Looks like this should be `... = {'name': component, ...`.

Reply:
it shouldn't :)
because component is used only for dockerlabel, which isn't used for source containers
and name is used for image names

Reply:
But in our mocked environment, `name` is `'ns/name'` and `component` is `'component'`. It's representing a Koji n-v-r of 'component-1.0-1', which ends up in the registry as ns/name:1.0-1.

We surely don't want to call the source container Koji build ns/name-source-1.0-1.

Reply:
ok